library/core/src/main/java/com/google/android/exoplayer2/text/SubtitleDecoderFactory.java

```diff
@@ -107,7 +107,7 @@ public SubtitleDecoder createDecoder(Format format) {
             case MimeTypes.APPLICATION_MP4CEA608:
               return new Cea608Decoder(format.sampleMimeType, format.accessibilityChannel);
             case MimeTypes.APPLICATION_CEA708:
-              return new Cea708Decoder(format.accessibilityChannel);
+              return new Cea708Decoder(format.accessibilityChannel, format.isWideAspectRatio);
```

Comment:
Feel free to just put `isWideAspectRatio` into `initializationData` for now, as suggested in `Format`. However the eventual goal would be to have these decoders just take `initializationData` like the others. We'll probably move `accessibilityChannel` and `packetLength` into `initializationData` in a subsequent change.

(`packetLength` is derived from `sampleMimeType`, which is why `sampleMimeType` is currently being passed to the 608 decoder)